src/monsters.h

```diff
@@ -181,8 +196,8 @@ class Monsters
 
 		std::map<std::string, uint32_t> monsterNames;
 		std::map<MonsterType*, std::string> monsterScriptList;
-		std::map<uint32_t, MonsterType*> monsters;
-		LuaScriptInterface* scriptInterface;
+		std::map<uint32_t, MonsterType> monsters;
```

Comment:
That's not a problem. I don't think MonsterType(id) was practical anyway, where'd you get the id from?